modules/etcd-deployment-caveats-span.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * etcd/etcd-guidance-span.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="deployment-caveats-span_{context}"]
+= Deployment caveats for spanned clusters
+
+The guidance provided in this documentation focuses on general aspects of cluster deployment that spans data centers. Some caveats to remember:
+
+* While the designs for deployments that span data centers are not bound by any special support requirements, these clusters do have additional inherent complexities that can require additional consideration or support involvement (time to identify, remediate and resolve issues) when compared to a standard single-site cluster.
```

Comment:
Per [ISG](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#w): Use while only to refer to a period of time. To avoid potential ambiguity, do not use the term as a synonym for although or though in documentation.

```suggestion
* Although the designs for deployments that span data centers are not bound by any special support requirements, these clusters do have additional inherent complexities that can require additional consideration or support involvement (time to identify, remediate, and resolve issues) when compared to a standard single-site cluster.
```